app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CoursesController < ApplicationController
```

Comment:
The Courses `module` will be empty, it's implemented in the other files.

What's de-coping?